brayns/engine/json/EngineFactory.h

```diff
@@ -62,11 +63,10 @@ class EngineFactory
         T deserialize(const JsonValue &payload) const override
         {
             auto schema = Json::getSchema<SubT>();
-            auto schemaErrors = JsonSchemaValidator::validate(payload, schema);
-            if (!schemaErrors.empty())
+            auto errors = Json::validate(payload, schema);
+            if (!errors.empty())
             {
-                auto errors = StringJoiner::join(schemaErrors, "\n");
-                throw std::invalid_argument("Cannot parse json: " + errors);
+                throw JsonSchemaException("Invalid engine JSON schema", std::move(errors));
```

Comment:
Personal perference, but looks weird having to move the list of errors to the exception. Similar issue as with the image decoder of previous PRs.